src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,28 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period "
+                + dateFrom + " - " + dateTo);
+        for (String array:names) {
```

Comment:
```suggestion
        for (String name : names) {
```

not informative variable name